code/contacts/org.eclipse.scout.contacts.client/src/main/java/org/eclipse/scout/contacts/client/Icons.java

```diff
@@ -19,7 +19,6 @@
   public static final String EclipseScout = "eclipse_scout";
   public static final String OrganizationLarge = "house";
   public static final String Organization = "font:awesomeIcons \uf015";
-  public static final String Person = "user";
```

Comment:
The constant Person was changed on 10.0 to User -> We should do the same here.
See https://github.com/BSI-Business-Systems-Integration-AG/org.eclipse.scout.docs/commit/ca34c4f17ca240035d2010d6a1700b828b8948e9

Reply:
Done.